data/publiccloud/terraform/azure_cloud-netconfig.tf

```diff
@@ -0,0 +1,289 @@
+terraform {
+  required_providers {
+    azurerm = {
+      version = "= 3.48.0"
+      source  = "hashicorp/azurerm"
+    }
+    random = {
+      version = "= 3.1.0"
+      source  = "hashicorp/random"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+variable "instance_count" {
+  default = "1"
+}
+
+variable "secondary_address" {
+  default = "2"
+}
+
+variable "name" {
+  default = "openqa-vm"
+}
+
+variable "type" {
+  default = "Standard_B2s"
+}
+
+variable "region" {
+  default = "westeurope"
+}
+
+variable "image_id" {
+  default = ""
+}
+
+variable "image_uri" {
+  default = ""
+}
+
+variable "publisher" {
+  default = "SUSE"
+}
+
+variable "offer" {
+  default = ""
+}
+
+variable "sku" {
+  default = "gen2"
+}
+
+variable "extra-disk-size" {
```

Comment:
if we creating dedicated profile we may cleaned up from things which are not needed for this certain test . we won't add extra disk here so let's drop quite some code from it